tasks/30_people-in-bus/people-in-bus.js

```diff
@@ -0,0 +1,6 @@
+const peopleInBus = function(busStops) {
+  return busStops.reduce((result, current) =>
+    result + current[0] - current[1], 0);
```

Comment:
Brackets positions seem odd to me. Please choose either you keep on the same line or move closing bracket on the next line